package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "map",
```

Comment:
Does this name have any influence on the directory that the plugin is installed to? My understanding is that this is important in how the primary plugin definition is loaded.